src/modules/map/components/LocationPopup.vue

```diff
@@ -1,45 +1,39 @@
 <script setup>
 /** Right click pop up which shows the coordinates of the position under the cursor. */
 
-import proj4 from 'proj4'
-import { computed, onMounted, ref, watch } from 'vue'
+// importing directly the vue component, see https://github.com/ivanvermeyen/vue-collapse-transition/issues/5
+import tippy from 'tippy.js'
+import { computed, onBeforeUnmount, onMounted, ref, watch } from 'vue'
 import { useI18n } from 'vue-i18n'
 import { useRoute } from 'vue-router'
 import { useStore } from 'vuex'
 
-import { requestHeight } from '@/api/height.api'
-import { generateQrCode } from '@/api/qrcode.api'
 import { createShortLink } from '@/api/shortlink.api'
-import { registerWhat3WordsLocation } from '@/api/what3words.api'
 import CesiumPopover from '@/modules/map/components/cesium/CesiumPopover.vue'
-import LocationPopupCopyInput from '@/modules/map/components/LocationPopupCopyInput.vue'
-import LocationPopupCopySlot from '@/modules/map/components/LocationPopupCopySlot.vue'
+import LocationPopupPosition from '@/modules/map/components/LocationPopupPosition.vue'
+import LocationPopupShare from '@/modules/map/components/LocationPopupShare.vue'
 import OpenLayersPopover from '@/modules/map/components/openlayers/OpenLayersPopover.vue'
-import {
-    LV03Format,
-    LV95Format,
-    MGRSFormat,
-    UTMFormat,
-    WGS84Format,
-} from '@/utils/coordinates/coordinateFormat'
-import { WGS84 } from '@/utils/coordinates/coordinateSystems'
 import log from '@/utils/logging'
 import { stringifyQuery } from '@/utils/url-router'
 
-const what3Words = ref(null)
-const height = ref(null)
-const qrCodeImageSrc = ref(null)
-const shareLinkUrlShorten = ref(null)
-const shareLinkUrl = ref(null)
-
 const i18n = useI18n()
-const route = useRoute()
 const store = useStore()
 
 const clickInfo = computed(() => store.state.map.clickInfo)
-const currentLang = computed(() => store.state.i18n.lang)
 const projection = computed(() => store.state.position.projection)
 const showIn3d = computed(() => store.state.cesium.active)
+const currentLang = computed(() => store.state.i18n.lang)
+
+const route = useRoute()
+const selectedTab = ref('position')
+const shareTabButton = ref(null)
+const newClickInfo = ref(true)
+const showEmbedSharing = ref(false)
```

Comment:
This could be simplified by using a computed property 
```
const showEmbedSharing = computed(()=> selectedTab.value === 'share')
```
So you don't have to set it and clear it below